framework/src/main/java/org/tron/core/Wallet.java

```diff
@@ -2054,8 +2054,16 @@ public IncrementalMerkleTree getMerkleTreeOfBlock(long blockNum) throws ZksnarkE
 
     try {
       if (chainBaseManager.getMerkleTreeIndexStore().has(ByteArray.fromLong(blockNum))) {
-        return IncrementalMerkleTree
-            .parseFrom(chainBaseManager.getMerkleTreeIndexStore().get(blockNum));
+        byte[] treeRoot = chainBaseManager.getMerkleTreeIndexStore().get(blockNum);
+        if (treeRoot == null) {
+          return null;
+        }
+        IncrementalMerkleTreeCapsule treeCapsule = chainBaseManager.getMerkleTreeStore()
+            .get(treeRoot);
+        if (treeCapsule == null) {
+          return null;
+        }
+        return treeCapsule.getInstance();
```

Comment:
Why do you need to modify the logic code in the PR for adding test cases?

Reply:
There is a bug in method getMerkleTreeOfBlock,  but this mehod is not invoked by any other class.